.chloggen/1418-decouple-openshift-logic-from-routes.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
```

Comment:
Is this a user-facing bug fix or fix of the e2e test? 

I would prefer to skip changelog for fixing tests

Reply:
Is not a test fix.
I cataloged it as bug fix because installing the OpenShift Routes in a Kubernetes cluster, will make the operator think it is an OpenShift server.